addon/adapters/application.js

```diff
@@ -128,7 +128,6 @@ export default Ember.Object.extend(FetchMixin, Evented, {
     // use resource's service if in container, otherwise use this service to fetch
     let owner = (typeof getOwner === 'function') ? getOwner(this) : this.container;
     let service = owner.lookup('service:' + pluralize(type)) || this;
-    url = this.fetchUrl(url);
```

Comment:
I'm pretty sure [this](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/adapters/application.js#L356) is that hook.